src/controller/ProductController.js

```
@@ -1269,7 +1269,7 @@ export const getBusinessAnalytics = async (req, res, next) => {
       {
         $project: {
           _id: 0,
-          gender: "$_id",
+          _id: "$_id",
```

Biome (1.9.4)

[error] This property value named _id is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named _id


(lint/suspicious/noDuplicateObjectKeys)